### PR TITLE
feat(YouTube/Overlay buttons): add patch option `Change top buttons`

### PR DESCRIPTION
**Pull request by MondayNitro**
add settings, captions icons for the 3 overlay button styles

### CODE REVIEW

**Comment by MondayNitro** [comment]
if someone can add code for make text bold in bold & rounded style (rounded is also thick like bold icons) then it will improve consistency


photo_6120568498812994071_w

change from `normal` to `bold` in youtube_controls_bottom_ui_container.xml

Screenshot 2024-07-22 152122

**Comment by MondayNitro** [comment]
also made round edges like other thin-type overlay buttons
xuCn8ux2gbs_248

**Comment by inotia00** [comment]
Overall good
PR is missing the copy command for the new icons added in the `Overlay buttons` patch

The `Overlay buttons` patch is basically a patch that adds new overlay buttons like the always repeat button, copy video URL button, etc

If you want it to be integrated into the `Overlay buttons` patch, it would be better to add a patch option

File: src/main/kotlin/app/revanced/patches/youtube/player/overlaybuttons/OverlayButtonsPatch.kt [hunk 1]
```diff
@@ -67,6 +68,15 @@ object OverlayButtonsPatch : BaseResourcePatch(
         required = true
     )
 
+    // Option to change top buttons
+    private val ChangeTopButtons by booleanPatchOption(
+        key = "ChangeTopButtons",
+        default = false,
+        title = "Change top buttons",
+        description = "Change the icons at the top of the player.",
```
Comment (inotia00) [inline src/main/kotlin/app/revanced/patches/youtube/player/overlaybuttons/OverlayButtonsPatch.kt:76]: @KobeW50 I'd like to request a review to ensure the description of the new patch options is adequate

Comment (inotia00) [reply src/main/kotlin/app/revanced/patches/youtube/player/overlaybuttons/OverlayButtonsPatch.kt:76]: I will merge the PR first
If changes are needed later, please open a new PR

**Comment by inotia00** [comment]
Thanks

**Comment by STEK1337** [comment]
> if someone can add code for make text bold in bold & rounded style (rounded is also thick like bold icons) then it will improve consistency
> 
> photo_6120568498812994071_w
> 
> change from `normal` to `bold` in youtube_controls_bottom_ui_container.xml
> 
> Screenshot 2024-07-22 152122

is this merged?

**Comment by inotia00** [comment]
No, I didn't merge it because I thought the patch would be too complicated

Even the first image is hard to find a connection with the Overlay buttons patch

**Comment by Aryan-Chauhan** [comment]
@MondayNitro Sorry to bother you, but can you also design Rounded, Bold & Thin icons for the `Collapse Button`?

**Pull request merged**